### PR TITLE
Team changes delivery

### PR DESCRIPTION
**Pull request by davethenipper**
My first pull request.  Let me know how it goes.

This delivery contains all the team changes.

@YashdalfTheGray @Mctalian

### CODE REVIEW

**Comment by YashdalfTheGray** [comment]
Your travis build failed. You've got some jshint and jscs errors.

File: src/ideas/ideasView/ideasView.js [hunk 1]
```diff
@@ -28,8 +28,11 @@ angular.module('flintAndSteel')
             $scope.searchText = undefined;
             ctrl.newComment = '';
             ctrl.newBack = '';
+            ctrl.newTeam = '';
             ctrl.enableEdit = false;
 
+            var ideaPromise = $q.defer();
```
Comment (YashdalfTheGray) [inline src/ideas/ideasView/ideasView.js]: What are you using this promise for?

File: src/ideas/ideasView/ideasView.spec.js [hunk 2]
```diff
@@ -348,4 +348,74 @@ describe('IdeasViewCtrl', function() {
...
+ 
+        var teamLength = 0;
+        var mockIdea; 
+        
+        beforeEach(function() {
+            loginSvcMock.checkLogin(authorAccount);
+            ideaSvcMock.getIdea(null, function(idea) {
+                mockIdea = idea;
+            });
+            // Zero out the array
```
Comment (YashdalfTheGray) [inline src/ideas/ideasView/ideasView.spec.js]: This is probably the most C++ comment I've even seen :+1:

File: src/ideas/ideasView/ideasView.js (hunk 1 above)
Comment (davethenipper) [inline src/ideas/ideasView/ideasView.js]: So, this breaks 28 of our tests without a promise.  I would rather deliver the code and create an issue on github

**Comment by davethenipper** [comment]
This will close issue 35

File: src/ideas/ideasView/ideasView.js [hunk 3]
```diff
@@ -86,7 +93,7 @@ angular.module('flintAndSteel')
 
             $scope.addNewInteraction = function addNewInteraction(type) {
                 var now = new Date().toISOString();
-                if (type === 'comments' || type === 'backs') {
+                if (type === 'comments' || type === 'backs' || type === 'team') {
```
Comment (Mctalian) [inline src/ideas/ideasView/ideasView.js]: You don't actually use this.

File: src/ideas/ideasView/ideasView.js [hunk 4]
```diff
@@ -102,6 +109,13 @@ angular.module('flintAndSteel')
                             types: $scope.selectedTypes
                         });
                     }
+                    else if (type === 'team') {
```
Comment (Mctalian) [inline src/ideas/ideasView/ideasView.js]: You don't actually use this.

File: src/ideas/ideasView/ideasView.js [hunk 5]
```diff
@@ -28,8 +28,25 @@ angular.module('flintAndSteel')
             $scope.searchText = undefined;
             ctrl.newComment = '';
             ctrl.newBack = '';
+            ctrl.newTeam = '';
```
Comment (Mctalian) [inline src/ideas/ideasView/ideasView.js]: This isn't used either.

**Comment by Mctalian** [comment]
Approved :+1:

**Comment by YashdalfTheGray** [comment]
Done and done, approved.

**Pull request merged**